packages/babel-plugin-proposal-optional-chaining/src/index.js

```diff
@@ -86,6 +86,10 @@ export default declare((api, options) => {
                 context = object;
               }
 
+              if (t.isSuper(context)) {
```

Comment:
Nit: We can put this with the `if else` above.

Reply:
@jridgewell Would you like for `t.isSuper` to be called twice in the `if else`? because context would be null if it wasnt memoized before, right?

Reply:
There should only be one check needed. The only way `context` would be a `Super` here is if it went into the `else` case (because if it when through the `if` case it would be an `AssignmentExpression`). So we need to do an `else if` (before the current `else`) to check if `t.isSuper(object)`.

Reply:
@jridgewell Alright, that makes sense. The changes are done now 😄